src/rpc/core_rpc_server.cpp

```diff
@@ -1990,6 +1990,35 @@ namespace cryptonote
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool core_rpc_server::on_get_quorum_list(const COMMAND_RPC_GET_QUORUM_LIST::request& req, COMMAND_RPC_GET_QUORUM_LIST::response& res)
+  {
+    PERF_TIMER(on_get_quorum_list);
+    bool r;
+
+    if (use_bootstrap_daemon_if_necessary<COMMAND_RPC_GET_QUORUM_LIST>(invoke_http_mode::JON, "/get_quorum_list", req, res, r))
+    {
+      return r;
+    }
+
+    std::array<std::string, 10> quorum_array;
```

Comment:
std::vector<> (easier if we change the quorum size later, e.g. I think Kee wants 9 or something)

Reply:
Fixed.